test/e2e/syncer/syncer_test.go

```diff
@@ -87,36 +87,6 @@ func TestSyncerLifecycle(t *testing.T) {
 	downstreamNamespaceName, err := shared.PhysicalClusterNamespaceName(nsLocator)
 	require.NoError(t, err)
 
-	// TODO(marun) The name mapping should be defined for reuse outside of the transformName method in pkg/syncer
-	secretName := "kcp-default-token"
-	configMapName := "kcp-root-ca.crt"
-
-	t.Logf("Waiting for downstream service account secret %s/%s to be created...", downstreamNamespaceName, secretName)
-	require.Eventually(t, func() bool {
-		_, err = downstreamKubeClient.CoreV1().Secrets(downstreamNamespaceName).Get(ctx, secretName, metav1.GetOptions{})
-		if apierrors.IsNotFound(err) {
-			return false
-		}
-		if err != nil {
-			t.Errorf("saw an error waiting for downstream service account secret %s/%s to be created: %v", downstreamNamespaceName, secretName, err)
-			return false
-		}
-		return true
-	}, wait.ForeverTestTimeout, time.Millisecond*100, "downstream service account secret %s/%s was not created", downstreamNamespaceName, secretName)
-
-	t.Logf("Waiting for downstream configmap %s/%s to be created...", downstreamNamespaceName, configMapName)
-	require.Eventually(t, func() bool {
-		_, err = downstreamKubeClient.CoreV1().ConfigMaps(downstreamNamespaceName).Get(ctx, configMapName, metav1.GetOptions{})
-		if apierrors.IsNotFound(err) {
-			return false
-		}
-		if err != nil {
-			t.Errorf("saw an error waiting for downstream configmap %s/%s to be created: %v", downstreamNamespaceName, configMapName, err)
-			return false
-		}
-		return true
-	}, wait.ForeverTestTimeout, time.Millisecond*100, "downstream configmap %s/%s was not created", downstreamNamespaceName, configMapName)
-
```

Comment:
these removals surprise me. They exist to get early feedback in the e2e about broken syncing. Don't we still sync objects?

Reply:
part of the discussion here: https://github.com/kcp-dev/kcp/pull/1127#discussion_r879525593

Reply:
See https://github.com/kcp-dev/kcp/pull/1127#discussion_r885907503. tl/dr: I disagree that these checks are useless. Instead make them non-fragile, but keep them. The machinery behind the scenes is heavy and if it breaks you are lost without having intermediate steps.